Sources/SmileID/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -46,6 +46,7 @@
 "Action.TakePhoto" = "Take Photo";
 "Action.UploadPhoto" = "Upload Photo";
 "Action.Skip" = "Skip back of ID";
+
```

Comment:
Is it possible to add comments on these spaces?

Similar to android strings

Reply:
Yep, `//` comments work fine